contents/stats_distributions.md

```diff
@@ -0,0 +1,159 @@
+## Distributions {#sec:stats_dist}
+
+Before defining distributions, we must talk about **probability**.
+
+**The probability of an event is a real number between 0 and 1, where, 0 indicates the impossibility of the event and 1 indicates the certainty of the event**.
+The greater the likelihood of an event, the more likely it is that the event will occur.
+A simple example is the tossing of a fair (impartial) coin.
+Since the coin is fair, both results ("heads" and "tails") are equally likely; the probability of "heads" is equal to the probability of "tails"; and since no other result is possible, the probability of "heads" or "tails" is $\frac{1}{2}$ (which can also be written as 0.5 or 50%).
+
+Regarding notation, **we define $A$ as an event and $P(A)$ as the probability of event $A$**, thus:
+
+$$ \{P(A) \in \mathbb{R} : 0 \leq P(A) \leq 1 \}. $$ {#eq:probability}
+
+This means the "probability of the event to occur is the set of all real numbers between 0 and 1; including 0 and 1".
+In addition, we have three axioms, originated from @kolmogorovFoundationsTheoryProbability1933:
+
+1. **Non-negativity**: For all $A$, $P(A) \geq 0$. Every probability is positive (greater than or equal to zero), regardless of the event.
+2. **Additivity**: For two mutually exclusive events $A$ and $B$ (cannot occur at the same time): $P(A) = 1 - P(B)$ and $P(B) = 1 - P(A)$.
+3. **Normalization**: The probability of all possible events $A_1, A_2, \dots$ must add up to 1: $\sum_{n \in \mathbb{N}} P(A_n) = 1$; or, in the case of a continuous variable, integrate to 1: $\int^\infty_{-\infty} f(x) dx = 1$.
+
+Now we are ready to talk about distributions.
+A **distribution is a function which maps real numbers to a probability**:
```

Comment:
Simply put: a distribution is a just a collection of scores (counts) in a variable(observations). Later,  a **probability distribution** is defined by a function that will map each observation to a given probability, or course after proper normalisation of the distribution.

Reply:
Thanks done in 03a33c71f3a147f7b31cc9cf8064fc867f6c30f1